api.planx.uk/server.ts

```diff
@@ -1,4 +1,5 @@
 import "isomorphic-fetch";
+import "express-async-errors";
```

Comment:
This addresses an issue we've been having with errors thrown in `planx-core`. 

Express doesn't natively catch async errors (docs: https://expressjs.com/en/guide/error-handling.html) which is likely the reason a number aren't "bubbling up" correctly from `planx-core` correctly.

Without this added, the currently tests fail (unless I wrap another try/catch block around `team.getIntegrations()`) - I'd expect that we will now see an uptick of API errors now being caught by Airbrake 🤞